test/uk/gov/hmrc/helptosavefrontend/controllers/RegisterControllerSpec.scala

```diff
@@ -67,7 +68,23 @@ class RegisterControllerSpec
     mockMetrics,
     mockAuditor,
     fakeApplication,
-    mockPagerDuty)(
+    mockPagerDuty,
+    Configuration("enable-early-cap-check" → false))(
+    crypto, mockEmailValidation, transformer) {
+    override lazy val authConnector = mockAuthConnector
+  }
+
+  val trueEarlyCapController: RegisterController = new RegisterController(
```

Comment:
do we need this `val` in scope for the whole spec, it makes the spec a bit messy. What you could do is something like this:
```scala
def newController(earlyCapCheck: Boolean): RegisterController = new RegisterController(...)
val controller = newController(false)
```
and in each spec where you need it true you can do
```
  "blah blah blah" in {
     val controller = newController(true)
     ...
  }
```

Reply:
done, thanks for the idea :)